src/store/whiteboarding/fabric-utils.ts

```diff
@@ -144,13 +144,7 @@ export function moveLayer(direction: string, state: any) {
   })
   const apiCall = () => {
     const uuids: string[] = _.map(p.$canvas.getObjects(), 'uuid')
-    updateWhiteboardElementsOrder(p.$socket.id, uuids, state.whiteboard.id).then(() => {
-      p.$canvas.discardActiveObject().requestRenderAll()
-      if (elements.length === 1) {
-        // When only a single item was selected, re-select it
-        p.$canvas.setActiveObject($_getCanvasElement(elements[0].uuid))
```

Comment:
The line above was brought over from the old implementation and I see no need for it.  In fact, the user might perceive some weirdness if s/he deselects an object and then the line above re-selects it.